File: news_server/news_server_go/entity/category.go
package entity

/*
厦门大学计算机专业 | 前华为工程师
专注《零基础学编程系列》  http://lblbc.cn/blog
包含：Java | 安卓 | 前端 | Flutter | iOS | 小程序 | 鸿蒙
公众号：蓝不蓝编程
*/
type Category struct {
	ID   uint64 `gorm:"primary_key;auto_increment" json:"id"`
	Name string `gorm:"type:varchar(50)" json:"name"`
}

func (Category) TableName() string {
	return "news_category"
}

File: news_server/news_server_nodejs/app/models/category.js
/**
 * 厦门大学计算机专业 | 前华为工程师
 * 专注《零基础学编程系列》  http://lblbc.cn/blog
 * 包含：Java | 安卓 | 前端 | Flutter | iOS | 小程序 | 鸿蒙
 * 公众号：蓝不蓝编程
 */
const { sequelize } = require('@core/db')
const { Model, DataTypes } = require('sequelize')
class Category extends Model {
}

Category.init({
  id: { type: DataTypes.TEXT, primaryKey: true },
  name: { type: DataTypes.TEXT }
}, {
  sequelize,
  modelName: 'news_category',
  tableName: 'news_category'
})

module.exports = {
  Category
}
